### PR TITLE
[Release]: Privacy policies

### PR DESCRIPTION
**Pull request by jgagne**
## What this PR does

- Updates all privacy policy pages to current language per aquent.com

### Preview Content

- https://courses.gymna.si/privacy-policy
- https://courses.gymna.si/privacy-policy/european-rights
- https://courses.gymna.si/privacy-policy/california-privacy-rights
- https://courses.gymna.si/privacy-policy/privacy-shield-notice

### CODE REVIEW

File: static-pages/privacy-policy/california-privacy-rights/index.html [hunk 1]
```diff
@@ -23,16 +23,16 @@ <h1>California Privacy Rights</h1>
...
+    <li><strong>Right to Know:</strong> You have a right to be informed about the categories of personal information we collect about you, the purpose for which it was collected, the sources from which it was collected and the categories of third parties with whom we share your personal information.</li>
+    <li><strong>We post this information in our Privacy Notice.</strong> The personal information we collect can be found in the <strong><a href="/privacy-policy#information-we-collect">“Information We Collect”</a></strong> section, the business or commercial purpose for which it is used can be found in the <strong><a href="/privacy-policy#use-of-collected-data">“Use of Collected Data”</a></strong> section, and the categories of third parties to whom we disclose your personal data can be found in the <strong><a href="/privacy-policy#disclosure-of-your-personal-data">“Disclosure of Personal Data”</a></strong> section.
```
Comment (rediris) [inline static-pages/privacy-policy/california-privacy-rights/index.html:27]: This needs a closing `<li>`

**Review by rediris — CHANGES_REQUESTED** [review]
See note about closing `li` tag

File: static-pages/privacy-policy/california-privacy-rights/index.html [hunk 2]
```diff
@@ -23,16 +23,16 @@ <h1>California Privacy Rights</h1>
...
       <ul>
-        <li>For additional information on the data we collect and the purposes for which it is collected, please click <a href="https://thegymnasium.com/privacy-policy#data-category">here</a>.</li>
+        <li>For additional information on the data we collect and the purposes for which it is collected, please click <a href="/privacy-policy#data-category">here</a>.</li>
       </ul>
     </li>
```
Comment (jgagne) [inline static-pages/privacy-policy/california-privacy-rights/index.html:31]: @rediris — The list item closes here, with a nested list within a list item.

Comment (rediris) [reply static-pages/privacy-policy/california-privacy-rights/index.html:31]: Ah, gotcha!

**Review by rediris — APPROVED** [review]

**Pull request merged**